src/pinns_pde_solve.jl

```diff
@@ -611,13 +667,15 @@ function get_argument(eqs,dict_indvars,dict_depvars)
         map(x -> first(x), f_vars)
     end
     args_ = map(vars) do _vars
-        map(var -> var.args[2:end] , _vars)
+        ind_args_ = map(var -> var.args[2:end], _vars)
+        unionarg_ = union(ind_args_...)        
```

Comment:
Nice catch! The challenge was that if I wrote `union()` it would not work in the case `[0., 0.]` but if I wrote `vcat()` it would not work for `[[:x, :y], [:y]]`

For now I wrote solved this with this line – it eliminates repeated symbols but not repeated numbers. 
```julia
args_ = map(vars) do _vars
        ind_args_ = map(var -> var.args[2:end], _vars)
        # unites ind_args_ into a single array where symbols cannot repeat themselves but numbers can
        vcat(unique([ind_arg for ind_arg in Iterators.flatten(ind_args_) if isa.(ind_arg, Symbol)]), [ind_arg for ind_arg in Iterators.flatten(ind_args_) if !isa.(ind_arg, Symbol)])
    end
```

But I still think this isn't right because even though the NNPDE tests work, the neural adapter tests lose some accuracy (the tests fail unless I raise rtol to 0.3). I will spend some time trying to understand what this should look like with heterogeneous support.